lib/carrierwave/mounter.rb

```diff
@@ -60,7 +60,7 @@ def cache_names
     end
 
     def cache_names=(cache_names)
-      return if not cache_names or cache_names == "" or uploaders.any?(&:cached?)
+      return if !cache_names || cache_names.blank? || uploaders.any?(&:cached?)
```

Comment:
This also applies to here.
https://github.com/carrierwaveuploader/carrierwave/pull/2388#discussion_r279695554

Reply:
Sorry to bother you again.

I fixed them.